js/background.js

```diff
@@ -1,29 +1,27 @@
+"use strict";
 chrome.runtime.onInstalled.addListener(() => {
-  chrome.contextMenus.create({
-      id: "promptPalette",
-      title: "Prompt Palette",
-      contexts: ["all"]
-  });
+    chrome.contextMenus.create({
+        id: "promptPalette",
+        title: "Prompt Palette",
+        contexts: ["all"]
+    });
 });
-
 chrome.contextMenus.onClicked.addListener((info, tab) => {
-  if (info.menuItemId === "promptPalette") {
-      openPopup();
-  }
+    if (info.menuItemId === "promptPalette") {
+        openPopup();
+    }
 });
-
 chrome.commands.onCommand.addListener((command) => {
-  if (command === "open_prompt_palette") {
-      openPopup();
-  }
+    if (command === "open_prompt_palette") {
+        openPopup();
+    }
 });
-
 function openPopup() {
-  chrome.windows.create({
-      url: chrome.runtime.getURL("popup.html"),
-      type: 'popup',
-      focused: true,
-      width: 400,
-      height: 600
-  });
+    chrome.windows.create({
+        url: chrome.runtime.getURL("popup.html"),
+        type: 'popup',
+        focused: true,
+        width: 400,
+        height: 600
+    });
```

Comment:
Here's the difference of the original background.js, and the one generated by the typescript compiler from `background.ts`